electron/types.ts

```diff
@@ -90,6 +90,7 @@ export interface GameStatus {
     | 'done'
     | 'canceled'
     | 'moving'
+    | 'changing'
```

Comment:
I think this is unnecessary because this operation should be instant since is just a read/write of a text file. 
That statuses I use for long operations, to give some feedback on the cards :)